### ISSUE TITLE
Usersnap Feedback - It's doing that annoying thing again where it replaces my previously selec[...]

### ISSUE THREAD

**Issue by ascott** [participedia/usersnaps#909]
**Sender**: scottowenfletcher@gmail.com
**Comment**: It's doing that annoying thing again where it replaces my previously selected values (gif to come)
[Open #603 in Usersnap Dashboard](https://usersnap.com/a/#/participedia-dev/p/participedia-net-9cb6d041/603)

<a href='https://usersnappublic.s3.amazonaws.com/2019-10-27/17-20/f348b2b4-8ee6-4ffa-b93a-b2ef0ad72eb8.png'>![Usersnap Feedback - It's doing that annoying thing again where it replaces my previously selec(...)](https://usersnappublic.s3.amazonaws.com/2019-10-27/17-20/505a6e46-bec3-4e69-8eea-6848244726f0.jpg)</a>
<a href='https://usersnappublic.s3.amazonaws.com/2019-10-27/17-20/f348b2b4-8ee6-4ffa-b93a-b2ef0ad72eb8.png'>Download original image</a>

**Browser**: Chrome 77 (Windows 10)
**Referer**: [https://participedia.net/case/6223/edit?full=1](https://participedia.net/case/6223/edit?full=1)
**Screen size**: 1280 x 720 **Browser size**: 1920 x 867

Powered by [usersnap.com](https://usersnap.com/?utm_source=github_entry&utm_medium=web&utm_campaign=product)

**Comment by scottofletcher** [comment]
![Selection bug again](https://user-images.githubusercontent.com/29512491/67637839-bada5180-f89b-11e9-9728-e8e549155aac.gif)

**Comment by jesicarson** [comment]
@scottofletcher did you try refreshing the page?

**Comment by scottofletcher** [comment]
yes, I always manage to get it working one way or the other, but this keeps happening and it really disrupts my workflow. plus, if it starts happening to other users, it might confuse them, or cause them to fill out a form incorrectly (or to give up on trying to fill out the form)
